lib/providers/quoters/RfqQuoter.ts

```diff
@@ -42,9 +48,15 @@ export class RfqQuoter implements Quoter {
       axios.get(`${this.serviceUrl}dutch-auction/nonce?address=${swapper}&chainId=${request.info.tokenInChainId}`), // should also work for cross-chain?
     ];
 
+    const getPortionResponse = await this.portionProvider.getPortion(
```

Comment:
are we duplicating this request now, once for classic quotes and once for dutch quotes? You might be able to refactor this as a `dependency` from the quote context to deduplicate the requests

Reply:
though if the latency / networking cost of duplicating the call is acceptable im ok leaving it as is for now as its def simpler